packages/cli/src/bases/Package.ts

```diff
@@ -0,0 +1,64 @@
+import { execSync } from "child_process";
+import fs from "fs";
+import path from "path";
+
+export namespace Package {
+  export type Manager = "npm" | "yarn" | "pnpm";
+
+  export const create = (input: {
+    projectName: string;
+    projectPath: string;
+  }): void => {
+    const packageJson = {
+      name: input.projectName,
+      version: "0.0.1",
+      type: "module",
+      scripts: {
+        build: "tsc",
+        dev: `ts-node ${input.projectName}/cli.ts`,
+        start: `node ${input.projectName}/cli.js`,
+      },
+    };
+
+    fs.writeFileSync(
+      path.join(input.projectPath, "package.json"),
+      JSON.stringify(packageJson, null, 2)
+    );
+    console.log("✅ package.json created");
+  };
+
+  export const installPackage =
+    (packageManager: Manager) =>
+    (input: { projectPath: string; services: string[] }) => {
+      const installCmd = (pkg: string) =>
```

Comment:
I think a switch-case statement would be more suitable in this situation

Reply:
As I know, our team avoid to use the switch statement. so I use if-else statement instead of ternary operator.